doc/api/fs.md

```diff
@@ -846,10 +870,21 @@ If `options` is a string, then it specifies the encoding. Example:
 fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 ```
 
-Any specified file descriptor has to have been opened for appending.
+The `file` may be specified as a numeric file descriptor that has been opened
+for appending (using `fs.open()` or `fs.openSync()`. It is important to note
+that the file descriptor will not be closed automatically.
```

Comment:
Suggestion: `It is important to note that the file descriptor will not be closed automatically` -> `The file descriptor will not be closed automatically.`